optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -519,94 +517,34 @@ def _from_transformers(
         save_dir = Path(save_dir).joinpath(model_id, subfolder)
         save_dir.mkdir(parents=True, exist_ok=True)
 
-        tokenizer = AutoTokenizer.from_pretrained(model_id)
-
-        framework = FeaturesManager.determine_framework(os.path.join(model_id, subfolder))
-        model_class = FeaturesManager.get_model_class_for_feature(cls.export_feature, framework)
-        model = model_class.from_pretrained(
+        model = TasksManager.get_model_from_task(
+            cls.export_feature,
```

Comment:
Can't comment above, but it seems that the attribute `force_download` is used nowhere, should we add it to `TasksManager.get_model_from_task`?

Reply:
Sure added the `force_download` to the `TasksManager.get_model_from_task`. 

About the deprecated code. The `ORTTrainer` is still uses the configs from the `optimum.onnx`. We could do the cleanup when we integrate `exporters` with training. I could work on this.

Reply:
Oh right `ORTTrainer`'s export still uses them. That will be great if you could also update them! Thank you @mht-sharma !